app/controllers/account/personal_details_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Account::PersonalDetailsController < ApplicationController
+  layout "admin_layout"
+
+  before_action :authenticate_user!
+  before_action :authorise_user
+
+  def show; end
+
+  def update_organisation
+    new_organisation_id = params[:user][:organisation_id]
+    new_organisation = Organisation.find(new_organisation_id)
+
+    if current_user.update(organisation_id: new_organisation_id)
```

Comment:
Similar to the comment about `#update_role` below, I think we should probably use `UserUpdate` here too. Surprisingly, we don't record a specific `EventLog` about changing organisation but I suspect that's an oversight.

I'm hopeful we can get rid of `UserUpdate` at some point but while it's still around I think we should probably consistently use it.

Reply:
I've switched to using `UserUpdate` and I don't think you were suggesting that I should, but FYI, I haven't added the missing event type